packages/server/src/api/controllers/row/external.ts

```diff
@@ -58,7 +58,7 @@ export async function patch(ctx: BBContext) {
   return handleRequest(Operation.UPDATE, tableId, {
     id: breakRowIdField(id),
     row: inputs,
-    includeSqlRelationships: IncludeRelationship.EXCLUDE,
+    includeSqlRelationships: IncludeRelationship.INCLUDE,
```

Comment:
Why has this changed?

Reply:
This is one of the causes of the bug. Because we were not fetching the SQL relationships, any edition request was trying to recreate them, given that if could "not find any existing relationship" matching the incoming request

Reply:
Makes sense, thanks!